src/ca/wolfram/beta/symath/OperationNode.java

```diff
@@ -0,0 +1,33 @@
+package ca.wolfram.beta.symath;
+
+import java.util.List;
+
+public abstract class OperationNode implements MathNode {
+
+    protected List<MathNode> children;
+
+    public OperationNode(List<MathNode> children) {
+        this.children = children;
+    }
+
+    @Override
+    public List<MathNode> getChildren() {
+        return children;
+    }
+
+    @Override
+    public boolean isConstant() {
+        for (MathNode n : children)
```

Comment:
better to put this in simplfy. and have it called during construction and what not